package exception_handling;

public class try_catch {
try{
	 // some code here
}catch(SomeException se){
	// some code here
	}finally{
		// some code here
	}
}
/*Under which three circumstances will the code on line 37 be executed? (Choose three.)
A. The instance gets garbage collected.
B. The code on line 33 throws an exception.
C. The code on line 35 throws an exception.
D. The code on line 31 throws an exception.
E. The code on line 33 executes successfully.
Answer: B, C, E

*/